cups/translate-attrs.go

```diff
@@ -293,6 +308,21 @@ func convertMarkers(printerTags map[string][]string) (*[]cdd.Marker, *cdd.Marker
 	return &markers, &states
 }
 
+//fixMarkers corrects some drivers' marker names/types where CUPS detects names/types with a comma
+// as two separate values. The second value of these pairs contain a space, so it's easy to detect.
+func fixMarkers(values []string) []string {
+	var newValues []string
+
+	for i := range values {
+		if i > 0 && len(values[i]) > 1 && values[i][0] == ' ' {
+			newValues[len(newValues)-1] = strings.Join([]string{newValues[len(newValues)-1], values[i]}, ",")
```

Comment:
Mostly because I was curious, I [benchmarked](https://gist.github.com/korylprince/705aa5ecb6564daa0623) `strings.Join` vs `fmt.Sprintf` vs simple adding and found:

1. Adding (`one + "," + two`) was fastest
2. `strings.Join([]string{one, two}, ",")` was second fastest, taking about 60% longer than adding
3. `fmt.Sprintf("%s,%s", one, two)` was slowest, taking 3.5-4x longer than adding

I don't think it's going to matter unless you have millions of printers, but perhaps that is why `strings.Join` was used orignally.